src/js/dataMapping/navigation/menuOptions.js

```diff
@@ -14,7 +14,7 @@ export const searchOptions = [
     {
         label: 'Keyword Search',
         url: '#/keyword_search',
-        enabled: kGlobalConstants.KEYWORD_AVAILABLE
+        enabled: true
     }
```

Comment:
@jonhill13 I too like to live dangerously...........

Reply:
LOL, we will need to delete that variable from dev, test, and prod config. But I thought since keyword search is live, we should live bold and `true`